File: app/src/main/java/com/android/keeper/recycle_items/TaskItem.java
package com.android.keeper.recycle_items;

public class TaskItem {
    private int imageResource;
    private int taskId;
    private String taskTitle;
    private String taskDetails;

    public TaskItem(int imageResource,int taskId, String taskTitle, String taskDetails) {
        this.imageResource = imageResource;
        this.taskId=taskId;
        this.taskTitle = taskTitle;
        this.taskDetails = taskDetails;
    }

    public int getImageResource() {
        return imageResource;
    }

    public String getTaskTitle() {
        return taskTitle;
    }

    public String getTaskDetails() {
        return taskDetails;
    }

    public int getTaskId() {
        return taskId;
    }

    public void setTaskTitle(String taskTitle) {
        this.taskTitle = taskTitle;
    }

    public void setTaskDetails(String taskDetails) {
        this.taskDetails = taskDetails;
    }
}

File: app/src/main/java/com/android/keeper/recycle_items/TaskItem.kt
package com.android.keeper.recycle_items

import android.support.test.InstrumentationRegistry
import android.support.test.runner.AndroidJUnit4
import org.junit.runner.RunWith

class TaskItem(private var imageResource: Int, private val taskId: Int, private var taskTitle: String?, private var taskDetails: String?, private var taskDone: Boolean) {
    fun getImageResource(): Int {
        return imageResource
    }

    fun getTaskTitle(): String? {
        return taskTitle
    }

    fun getTaskDetails(): String? {
        return taskDetails
    }

    fun getTaskId(): Int {
        return taskId
    }

    fun setTaskTitle(taskTitle: String?) {
        this.taskTitle = taskTitle
    }

    fun setTaskDetails(taskDetails: String?) {
        this.taskDetails = taskDetails
    }

    fun setImageResource(imageResource: Int) {
        this.imageResource = imageResource
    }

    fun isTaskDone(): Boolean {
        return taskDone
    }

    fun setTaskDone(taskDone: Boolean) {
        this.taskDone = taskDone
    }

}